tests/integration/ovs_test.py

```diff
@@ -103,3 +117,44 @@ def test_create_and_remove_ovs_bridge_with_internal_port_and_static_ip():
 
     assertlib.assert_absent(BRIDGE1)
     assertlib.assert_absent(PORT1)
+
+
+@pytest.mark.xfail(raises=NmstateLibnmError, strict=True)
```

Comment:
Since we are expecting the exception, this should use `pytest.raises` in the test instead of expecting the whole test to fail because of an unhanded exception.

Reply:
Done! Thanks :-)